catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/less/styles.less

```diff
@@ -253,3 +254,43 @@ table.align-middle{
   }
 }
 
+.advanced-section {
+  display: none;
+}
+.form-segments {
+  margin-left: 10px;
+}
+
+.pop-div,.popover-content{
+  overflow-wrap: break-word;
+}
+
+input[type=number]::-webkit-inner-spin-button,
+input[type=number]::-webkit-outer-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+.inline-small {
```

Comment:
We are not always good at this yet but in general you want your classes to be based on semantics instead of presentation.

https://css-tricks.com/semantic-class-names/